From ba84c4aba56420315973026288e7b336006336a9 Mon Sep 17 00:00:00 2001
From: edef <edef@edef.eu>
Date: Tue, 30 Jul 2024 03:14:37 +0000
Subject: [PATCH] tests/erase: parenthesize braced initializer

memcpy is a macro on Darwin, and thus requires braced initializer lists
to be parenthesized.
---
 tests/erase_func_algo.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/tests/erase_func_algo.c b/tests/erase_func_algo.c
index 576923f8..b13ee70c 100644
--- a/tests/erase_func_algo.c
+++ b/tests/erase_func_algo.c
@@ -1233,13 +1233,13 @@ static void erase_unwritable_regions_skipflag_on_test_success(void **state)
 
 	// This test needs special block erase to emulate protected regions.
 	memcpy(g_test_erase_injector,
-		(erasefunc_t *const[]){
+		((erasefunc_t *const[]){
 			block_erase_chip_with_protected_region_1,
 			block_erase_chip_with_protected_region_2,
 			block_erase_chip_with_protected_region_3,
 			block_erase_chip_with_protected_region_4,
 			block_erase_chip_with_protected_region_5,
-		},
+		}),
 		sizeof(g_test_erase_injector)
 	);
 
@@ -1322,13 +1322,13 @@ static void write_unwritable_regions_skipflag_on_test_success(void **state) {
 
 	// This test needs special block erase to emulate protected regions.
 	memcpy(g_test_erase_injector,
-		(erasefunc_t *const[]){
+		((erasefunc_t *const[]){
 			block_erase_chip_with_protected_region_1,
 			block_erase_chip_with_protected_region_2,
 			block_erase_chip_with_protected_region_3,
 			block_erase_chip_with_protected_region_4,
 			block_erase_chip_with_protected_region_5,
-		},
+		}),
 		sizeof(g_test_erase_injector)
 	);
 
-- 
2.45.2

